### PR TITLE
[bitnami/influxdb] Add aws backup upload provider

### LINKED ISSUE #5388: InfluxDB s3 backup option
**Which chart**:
influxdb

**Is your feature request related to a problem? Please describe.**
Add a back-up upload provider for uploading to a S3 bucket. 

**Describe the solution you'd like**
The same as the Azure/Google backup cronjob script but use an AWS cli container to upload to S3. 

**Describe alternatives you've considered**
Maybe there is some other way that this is already possible, but that I could not find. 

**Additional context**
See the chart supplied by InfluxData: 
https://github.com/influxdata/helm-charts/blob/master/charts/influxdb/templates/backup-cronjob.yaml#L128

### PR DESCRIPTION
**Pull request by pixelfields**
**Description of the change**

Add AWS backup upload provider. Also backup script had to be fixed for InfluxDB version 1.x.

**Benefits**

InfluxDB backups can be uploaded to AWS S3 bucket.

**Possible drawbacks**

None

**Applicable issues**

  - fixes #5388

**Additional information**

N/A

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/influxdb/templates/influxdb/configmap-backup.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ data:
             echo "backuping ${DATABASE} db to {{ .Values.backup.directory }}/${DATABASE}"
             mkdir -p {{ .Values.backup.directory }}/${DATABASE}
 
-            influxd backup -host "${host}" -port {{ .Values.influxdb.service.port }} -portable -db ${DATABASE} {{ .Values.backup.directory }}/"${DATABASE}/${DATE}"
+            influxd backup -host "${host}:{{ .Values.influxdb.service.rpcPort }}" -portable -db ${DATABASE} {{ .Values.backup.directory }}/"${DATABASE}/${DATE}"
```
Comment (miguelaeh) [inline bitnami/influxdb/templates/influxdb/configmap-backup.yaml:46]: Could you explain why this change is needed? Weren't the `service.port` the correct port to perform the backup?

File: bitnami/influxdb/Chart.yaml [hunk 2]
```diff
@@ -24,4 +24,4 @@ name: influxdb
 sources:
   - https://github.com/bitnami/bitnami-docker-influxdb
   - https://www.influxdata.com/products/influxdb-overview/
-version: 2.5.0
+version: 2.5.1
```
Comment (miguelaeh) [inline bitnami/influxdb/Chart.yaml:27]:
```suggestion
version: 2.6.0
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you for the PR! Please take a look to the comments

File: bitnami/influxdb/templates/influxdb/configmap-backup.yaml (hunk 1 above)
Comment (pixelfields) [reply bitnami/influxdb/templates/influxdb/configmap-backup.yaml:46]: @miguelaeh Thanks for the comment! With the service port it simply doesn't work, as the remote connection is used here, and backups and restore is done by RPC port. I guess it hasn't been never tested before. The change is according the documentation here: 

https://docs.influxdata.com/influxdb/v1.8/administration/ports/#8088

https://docs.influxdata.com/influxdb/v1.8/administration/backup_and_restore/#online-backup-and-restore-for-influxdb-oss

> The default port used by the RPC service for RPC calls made by the CLI for backup and restore operations

File: bitnami/influxdb/Chart.yaml (hunk 2 above)
Comment (pixelfields) [reply bitnami/influxdb/Chart.yaml]: It's fixed now.

File: bitnami/influxdb/templates/influxdb/configmap-backup.yaml (hunk 1 above)
Comment (miguelaeh) [reply bitnami/influxdb/templates/influxdb/configmap-backup.yaml:46]: perfect! 👍

**Review by miguelaeh — APPROVED** [review]
LGTM!
Thank you very much!!
